homework/index.js

```diff
@@ -36,7 +36,44 @@
       if (err) {
         createAndAppend('div', root, { text: err.message, class: 'alert-error' });
       } else {
-        createAndAppend('pre', root, { text: JSON.stringify(data, null, 2) });
+        //createAndAppend('pre', root, { text: JSON.stringify(data, null, 2) });
+        const select = createAndAppend('select', root);
+        createAndAppend('option', select, { text: 'Click here to choose a Repository' });
```

Comment:
You've missed the requirement: At start-up your application should display information about the first repository as displayed in the select element. It's fine for this homework, but please add it for week2 homework.